.npmrc

```diff
@@ -1,4 +1,4 @@
-engine-strict=true
+engine-strict=false
```

Comment:
in order to allow this repo to upgrade to latest pnpm, needed to set to this to false. It appears PNPM is enforcing the version of PNPM for every transitive dependency.